drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_posting.rb

```diff
@@ -164,6 +166,23 @@ def track_status_changes
       end
     end
 
+    after_save :mark_household_members_dirty
```

Comment:
This will run sometimes unnecessarily, since my understanding is we don't strictly need to update CE client lists when the status changes internally, from assigned to accepted pending (for example). I think that seems acceptable, but let me know if you want to get more specific here?